explore_amy.ipynb

```diff
@@ -18,12 +18,11 @@
     "In the Cal-Adapt: Analytics Engine, there are three options for investigating AMY:  \n",
     "1. Absolute AMY: represents the historical reference period (1981-2010) or a future reference period based around a 30-year warming level time window. **Note: Absolute AMY data and display is not bias corrected, and should be used for visual data inspection only.**\n",
     "2. Difference AMY: represents the difference between a designated future 30-year period and the historical baseline (1981-2010)\n",
-    "3. *Coming soon!* Severe meteorological year: represents the difference between a designated extreme percentile for a designated future 30-year period and the historical baseline (1981-2010)\n",
+    "3. Severe meteorological year: represents the difference between the 90th percentile for a designated future 30-year period and the historical baseline (1981-2010)\n",
     "\n",
-    "For any given location, we can calculate the desired AMY option and display one of the following three options depending on your selections:\n",
+    "For any given location, we can calculate the desired AMY option and display one of the following options depending on your selections:\n",
     "1. Heatmap of AMY\n",
-    "2. Diurnal cycle of a typical day within the AMY\n",
-    "3. Annual cycle of a typical hour within the AMY"
+    "2. A typical month of choice within the AMY"
```

Comment:
I think it might be nice instead to clarify here what the panel GUI can do, vs. what the notebook shows you how to do in code. For example, "Using the handy-dandy app.explore.amy() panel, you can easily set how you want to compute the average meteorological year and view a heatmap of the resulting average meteorological year. Later, we'll break down the calculation and show you how to code it up yourself to output a typical month of choice within an average meteorological year". Something like that.

Reply:
I like this! I'll make the switch now